releases/release-5.3.4.md

```diff
@@ -0,0 +1,84 @@
+    - Fix the issue that sql returns `ERROR 1105 (HY000): close of nil channel` when using HashJoinExec [#30289](https://github.com/pingcap/tidb/issues/30289)
+    - Fix the different return type of date_add and date_sub between TiDB and MySQL. Change date_add and date_sub string_(int/string/real/decimal) function return type to string [#27573](https://github.com/pingcap/tidb/issues/27573)
+    - Fix the problem that tikv and tiflash return different results when query logical operations [#37258](https://github.com/pingcap/tidb/issues/37258)
+
+    <!--transaction owner: @cfzjywxk-->
+
+    - (dup) Fix the issue that the `EXPLAIN ANALYZE` statement with DML executors might return result before the transaction commit finishes [#37373](https://github.com/pingcap/tidb/issues/37373)
+    - Fix the issue that the region cache after merging many regions is not cleared properly [#37174](https://github.com/pingcap/tidb/issues/37174)
+
+    <!--planner owner: @qw4990-->
+
+    - (dup) Fix the issue that the EXECUTE might throw an unexpected error in specific scenarios [#37187](https://github.com/pingcap/tidb/issues/37187)
+    - (dup) Fix the issue that `GROUP CONCAT` with `ORDER BY` might fail when the `ORDER BY` clause contains a correlated subquery [#18216](https://github.com/pingcap/tidb/issues/18216)
+    - Fix the issue that wrong length and width are set for Decimal and Real when using plan-cache [#29565](https://github.com/pingcap/tidb/issues/29565)
+
++ PD
+
+    <!--owner: @nolouch-->
+
+    - (dup) Fix the issue that PD cannot correctly handle dashboard proxy requests [#5321](https://github.com/tikv/pd/issues/5321)
+    - (dup) Fix the issue that the TiFlash learner replica might not be created in specific scenarios [#5401](https://github.com/tikv/pd/issues/5401)
+    - (dup) Fix inaccurate Stream timeout and accelerate leader switchover [#5207](https://github.com/tikv/pd/issues/5207)
+
++ TiFlash
+
+    <!--compute owner: @zanmato1984-->
+
+    - Fix the issue that logical operators return wrong results when the argument type is UInt8  [#6127](https://github.com/pingcap/tiflash/issues/6127)
+
+    <!--storage owner: @flowbehappy-->
+
+    - Fix the issue that TiFlash crash due to using `0.0` as the integer's default value. E.g. `i` int(11) NOT NULL DEFAULT '0.0' [#3157](https://github.com/pingcap/tiflash/issues/3157)
+
++ Tools
+
+    + Dumpling
+
+    <!--owner: @niubell-->
+
+        - Fix the issue that dumpling can't dump with `--compress` option and s3 output directory [#30534](https://github.com/pingcap/tidb/issues/30534)
+
+    + TiCDC
+
+    <!--owner: @nongfushanquan-->
+
+        - Fix a issue that causes MySQL related error not reported to owner in time [#6698](https://github.com/pingcap/tiflow/issues/6698)
```

Comment:
```suggestion
        - Fix the issue that changefeed state is incorrect because a MySQL-related error is not reported to the owner in time [#6698](https://github.com/pingcap/tiflow/issues/6698)
```